src/client/package.json

```diff
@@ -40,7 +40,11 @@
     "webpack-cli": "^3.3.7",
     "webpack-dev-server": "^3.8.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "semantic-ui-css": "^2.4.1",
+    "semantic-ui-react": "^0.88.1",
+    "url-loader": "^2.1.0"
```

Comment:
all 3 of these should be "devDependencies"

Reply:
Done